attiny/attiny.ino

```diff
@@ -15,12 +15,14 @@
 #define MINUTE_COUNTDOWN 108 // = (60/0.5)*90% as each interupt occurs every 0.5 seconds. -10% because of inaccurate internal clock
 #define PI_WDT_RESET_VAL 30000 // 5 minutes 100*60*5
 
-
-int piPowerOffWait = PI_POWER_OFF_WAIT_TIME;
-uint16_t piSleepTime = 0; // Counting down the time until the pi will be turned on in minutes. If this is 0 the pi will be powered on.
+volatile uint16_t piSleepTime = 0; // Counting down the time until the pi will be turned on in minutes. If this is 0 the pi will be powered on.
```

Comment:
I know it's annoying but whenever `piSleepTime`, `blinks` and `minuteCountdown` get read or written outside of an interrupt handler you need to wrap that bit of code in `cli() and `sei()` calls. Not doing so will lead to occasional strange behaviour.

It looks like that `blinks` only holds small values so you could avoid the issue their by making it a `byte` or `uint8_t` (if that exists). (I suspect an int is 16 bits on an attiny85 but I'm not sure)